src/lib/isStrongPassword.js

```diff
@@ -8,6 +8,7 @@ const symbolRegex = /^[-#!$@%^&*()_+|~=`{}\[\]:";'<>?,.\/ ]$/;
 
 const defaultOptions = {
   minLength: 8,
+  maxLength: 15,
```

Comment:
15 characters as a default seem to be few because:
- Password must be hashed when stored, so their length doesn't matter
- Some browser password password generators (like Safari) automatically create passwords longer than 15 characters and is always annoying to manually edit those.

So my suggestion is to avoid a default value for this option or to increase it at least to 20/25

Reply:
I totally agree with @fedeci